src/VrRetreat.WebApp/Controllers/AccountController.cs

```diff
@@ -108,4 +108,100 @@ public async Task<IActionResult> Logout()
         await _signInManager.SignOutAsync();
         return RedirectToAction(nameof(HomeController.Index), "Home");
     }
+
+    [HttpGet]
+    public IActionResult Settings()
+    {
+        if (!_signInManager.IsSignedIn(User))
+        {
+            return RedirectToAction("Index", "Home");
+        }
+        
+        return View();
+    }
+
+    [HttpPost]
+    [ValidateAntiForgeryToken]
+    public async Task<IActionResult> ResetPassword(UserSettingsModel settingsModel)
+    {
+        if (!ModelState.IsValid)
+        {
+            return View(nameof(Settings));
+        }
+
+        var currentUser = await _userManager.GetUserAsync(HttpContext.User);
```

Comment:
That's neat, I made a get by username method like a fool. Didn't realize I could use the usermanager. 😅 

_The more you know_